README.md
TYPO3 Extensions
================

![Status](https://img.shields.io/badge/status-obsolete-lightgray.svg?style=flat)
![License](https://img.shields.io/badge/license-gpl2-brightgreen.svg?style=flat)
[![Contact](https://img.shields.io/badge/contact-@macmade-blue.svg?style=flat)](https://twitter.com/macmade)

Repository Infos
----------------

    Owner:			Jean-David Gadina - XS-Labs
    Web:			www.xs-labs.com
    Blog:			www.noxeos.com
    Twitter:		@macmade
    GitHub:			github.com/macmade
    LinkedIn:		ch.linkedin.com/in/macmade/
    StackOverflow:	stackoverflow.com/users/182676/macmade


oop/ChangeLog
#
# ChangeLog for /trunk/oop
# 
# Generated by Trac 0.11
# 02/06/09 20:02:58
 
Fri, 06 Feb 2009 18:56:03 GMT macmade [914]
    * trunk/oop/classes/tx/oop/String/Utils.class.php (modified)

    Added a __get() method for the ASCII control characters.


Fri, 06 Feb 2009 18:48:23 GMT macmade [913]
    * trunk/oop/classes/tx/oop/Module/Base.class.php (modified)
    * trunk/oop/classes/tx/oop/Plugin/Base.class.php (modified)

    Cleaning...


Fri, 06 Feb 2009 18:18:27 GMT macmade [912]
    * trunk/oop/classes/tx/oop/String (added)
    * trunk/oop/classes/tx/oop/String/Utils.class.php (added)

    Added a string utility class.


Fri, 06 Feb 2009 17:06:45 GMT macmade [910]
    * trunk/oop/classes/tx/oop/Debug (added)
    * trunk/oop/classes/tx/oop/Debug/Tools.class.php (added)
    * trunk/oop/classes/tx/oop/Typo3/Utils.class.php (modified)

    The debug methods are now in a separate class.


Fri, 06 Feb 2009 17:04:44 GMT macmade [909]
    * trunk/oop/classes/tx/oop/Typo3/Utils.class.php (modified)

    Cleaning...


Fri, 06 Feb 2009 17:03:34 GMT macmade [908]
    * trunk/oop/classes/tx/oop/Typo3/Utils.class.php (modified)

    Added debug methods.


Fri, 06 Feb 2009 17:03:22 GMT macmade [907]
    * trunk/oop/classes/tx/oop/Plugin/Base.class.php (modified)

    Bugfix


Fri, 06 Feb 2009 14:48:22 GMT macmade [901]
    * trunk/oop/classes/tx/oop/Plugin/Base.class.php (modified)

    Cleaning...


Fri, 06 Feb 2009 14:46:50 GMT macmade [900]
    * trunk/oop/classes/tx/oop/Lang/Getter.class.php (modified)
    * trunk/oop/classes/tx/oop/Plugin/Base.class.php (modified)

    Bugfix


Fri, 06 Feb 2009 14:41:37 GMT macmade [899]
    * trunk/oop/classes/tx/oop/Plugin/Wizard/Icon.class.php (modified)
    * trunk/oop/classes/tx/oop/Typo3/Utils.class.php (modified)

    Bugfix...


Fri, 06 Feb 2009 13:53:31 GMT macmade [898]
    * trunk/oop/classes/tx/oop/Typo3/Utils.class.php (added)

    Added the TYPO3 utilities class.


Fri, 06 Feb 2009 01:15:50 GMT macmade [891]
    * trunk/oop/classes/tx/oop/Plugin/Wizard (added)
    * trunk/oop/classes/tx/oop/Plugin/Wizard/Icon.class.php (added)

    Added the abstract class for the plugin wizard icon classes.


Fri, 06 Feb 2009 01:15:16 GMT macmade [890]
    * trunk/oop/classes/tx/oop/Plugin/Base.class.php (modified)

    Added the properties needed by tslib_piBase.


Fri, 06 Feb 2009 00:28:17 GMT macmade [889]
    * trunk/oop/classes/tx/oop/Module/Data.class.php (modified)

    The backend module data class now implements the ArrayAccess and
    Iterator interfaces.


Fri, 06 Feb 2009 00:14:42 GMT macmade [888]
    * trunk/oop/classes/tx/oop/Module/Base.class.php (modified)
    * trunk/oop/classes/tx/oop/Module/Data.class.php (added)

    Added a class for the backend module data.


Fri, 06 Feb 2009 00:14:06 GMT macmade [887]
    * trunk/oop/classes/tx/oop/Core/ClassManager.class.php (modified)

    Cleaning...


Thu, 05 Feb 2009 23:13:57 GMT macmade [886]
    * trunk/oop/classes/tx/oop/Typo3/ClassManager.class.php (modified)

    Cleaning...


Thu, 05 Feb 2009 23:04:52 GMT macmade [885]
    * trunk/oop/classes/tx/oop/Typo3/ClassManager.class.php (modified)

    Bugfix...


Thu, 05 Feb 2009 22:46:30 GMT macmade [884]
    * trunk/oop/classes/tx/oop/Typo3/ClassManager.class.php (modified)

    Cleaning...


Thu, 05 Feb 2009 22:44:55 GMT macmade [883]
    * trunk/oop/classes/tx/oop/Typo3 (added)
    * trunk/oop/classes/tx/oop/Typo3/ClassManager (added)
    * trunk/oop/classes/tx/oop/Typo3/ClassManager.class.php (added)
    * trunk/oop/classes/tx/oop/Typo3/ClassManager/Exception.class.php (added)
    * trunk/oop/ext_localconf.php (modified)

    Added an SPL autoload class for the TYPO3 classes (core and third-
    party extensions).


Thu, 05 Feb 2009 22:21:44 GMT macmade [882]
    * trunk/oop/classes/tx/oop/Core/ClassManager.class.php (modified)
    * trunk/oop/ext_localconf.php (modified)

    Cleaning...


Thu, 05 Feb 2009 22:09:15 GMT macmade [881]
    * trunk/oop/ext_conf_template.txt (modified)
    * trunk/oop/ext_localconf.php (modified)

    Added an option to auto-load classes from tslib or t3lib.


Thu, 05 Feb 2009 22:08:45 GMT macmade [880]
    * trunk/oop/ext_conf_template.txt (modified)
    * trunk/oop/ext_localconf.php (modified)

    Added an option to enable/disable to XHTML formatting.


Thu, 05 Feb 2009 21:14:14 GMT macmade [878]
    * trunk/oop/classes/tx/oop/Flexform/Field/Section/Element/Field (added)
    * trunk/oop/classes/tx/oop/Flexform/Field/Section/Element/Field/Check.class.php (added)
    * trunk/oop/classes/tx/oop/Flexform/Field/Section/Element/Field/Group.class.php (added)
    * trunk/oop/classes/tx/oop/Flexform/Field/Section/Element/Field/Inline.class.php (added)
    * trunk/oop/classes/tx/oop/Flexform/Field/Section/Element/Field/Input.class.php (added)
    * trunk/oop/classes/tx/oop/Flexform/Field/Section/Element/Field/None.class.php (added)
    * trunk/oop/classes/tx/oop/Flexform/Field/Section/Element/Field/Passthrough.class.php (added)
    * trunk/oop/classes/tx/oop/Flexform/Field/Section/Element/Field/Radio.class.php (added)
    * trunk/oop/classes/tx/oop/Flexform/Field/Section/Element/Field/Select.class.php (added)
    * trunk/oop/classes/tx/oop/Flexform/Field/Section/Element/Field/Text.class.php (added)
    * trunk/oop/classes/tx/oop/Flexform/Field/Section/Element/Field/User.class.php (added)

    Added the flexform section element field classes.


Thu, 05 Feb 2009 21:13:36 GMT macmade [877]
    * trunk/oop/classes/tx/oop/Flexform/Field/Section/Element (added)
    * trunk/oop/classes/tx/oop/Flexform/Field/Section/Element.class.php (added)
    * trunk/oop/classes/tx/oop/Flexform/Field/Section/Element/Exception.class.php (added)
    * trunk/oop/classes/tx/oop/Flexform/Field/Section/Element/Interface.class.php (added)

    Added the flexform section element class, with it's interface and
    exception class.


Thu, 05 Feb 2009 21:12:26 GMT macmade [876]
    * trunk/oop/classes/tx/oop/Flexform/Field/Section (added)

cjf/ext_emconf.php
<?php

########################################################################
# Extension Manager/Repository config file for ext: "cjf"
#
# Auto generated 20-11-2006 09:08
#
# Manual updates:
# Only the data in the array - anything else is removed by next write.
# "version" and "dependencies" must not be touched!
########################################################################

$EM_CONF[$_EXTKEY] = array(
	'title' => 'Cully Jazz Festival',
	'description' => '',
	'category' => 'plugin',
	'author' => 'Jean-David Gadina',
	'author_email' => 'info@macmade.net',
	'shy' => '',
	'dependencies' => 'cms,lang,api_macmade,fpdf',
	'conflicts' => '',
	'priority' => '',
	'module' => 'mod1',
	'state' => 'beta',
	'internal' => '',
	'uploadfolder' => 1,
	'createDirs' => 'uploads/tx_cjf/rte/',
	'modify_tables' => '',
	'clearCacheOnLoad' => 1,
	'lockType' => '',
	'author_company' => 'macmade.net',
	'version' => '0.0.0',
	'constraints' => array(
		'depends' => array(
			'cms' => '',
			'lang' => '',
			'api_macmade' => '0.2.8-',
			'fpdf' => '',
		),
		'conflicts' => array(
		),
		'suggests' => array(
		),
	),
	'_md5_values_when_last_written' => 'a:23:{s:9:".DS_Store";s:4:"4b1a";s:12:"ext_icon.gif";s:4:"c839";s:17:"ext_localconf.php";s:4:"8bd8";s:14:"ext_tables.php";s:4:"48e8";s:14:"ext_tables.sql";s:4:"93f3";s:24:"ext_typoscript_setup.txt";s:4:"34c7";s:22:"flexform_ds_groups.xml";s:4:"cb6d";s:19:"flexform_ds_pi1.xml";s:4:"58a8";s:16:"locallang_db.xml";s:4:"ac39";s:7:"tca.php";s:4:"bc31";s:24:"pi1/class.tx_cjf_pi1.php";s:4:"d52a";s:17:"pi1/locallang.xml";s:4:"4033";s:13:"res/.DS_Store";s:4:"d043";s:27:"res/icon_tx_cjf_artists.gif";s:4:"b2ec";s:30:"res/icon_tx_cjf_artists__h.gif";s:4:"25d7";s:26:"res/icon_tx_cjf_events.gif";s:4:"feb6";s:29:"res/icon_tx_cjf_events__h.gif";s:4:"25d7";s:26:"res/icon_tx_cjf_groups.gif";s:4:"8c2d";s:29:"res/icon_tx_cjf_groups__h.gif";s:4:"25d7";s:26:"res/icon_tx_cjf_places.gif";s:4:"ad88";s:29:"res/icon_tx_cjf_places__h.gif";s:4:"25d7";s:26:"res/icon_tx_cjf_styles.gif";s:4:"48b0";s:29:"res/icon_tx_cjf_styles__h.gif";s:4:"25d7";}',
	'suggests' => array(
	),
);

?>
